pkg/storage/image.go

```diff
@@ -285,11 +328,19 @@ func GetImageService(store storage.Store, defaultTransport string, insecureRegis
 		}
 	}
 
+	if len(additionalRegistries) == 0 {
+		// TODO(runcom): make docker.io a CONST
+		additionalRegistries = []string{"docker.io"}
```

Comment:
indeed, I will change the config file to have "docker.io" in it by default so that we don't hardcode docker.io